docker/python3-ubi/Dockerfile

```diff
@@ -19,11 +19,13 @@ RUN dnf upgrade -y --nodocs && \
     dnf clean all && \
     rm -rf /var/cache/dnf
 
-# install python3
-RUN wget https://www.python.org/ftp/python/3.10.8/Python-3.10.8.tgz && \
+ARG PYTHON_VER=3.10.8
```

Comment:
any way to declare as an ENV here and use it on line 27 so we dont need to declare 3.10.8 twice? 
Id also use a symlink to the dynamic_version script we have in python3 so we dont need to repeat 3.10.8 in the build conf either

Reply:
we use multistage, is it possible? regarding dynamic_version this can be done

Reply:
added dynamic version symlink